script/Common.s.sol

```diff
@@ -220,6 +246,20 @@ abstract contract Common is Contracts, Params {
         );
   }
 
+  function deployTokenDistributor() public updateParams {
+    // Deploy aidrop distributor contract
+    tokenDistributor = new TokenDistributor({
+    _root: bytes32(0), // TODO: add merkle root
+    _token: ERC20Votes(address(protocolToken)),
+    _totalClaimable: 1_000_000e18, // TODO: add to params and test
+    _claimPeriodStart: block.timestamp + 1, // TODO: use block.timestamp + x hs
+    _claimPeriodEnd: block.timestamp + 864000 // TODO: use block.timestamp + 1 month
```

Comment:
Not sure we want to limit to 1 month

Reply:
let's remember to define them when setting all initial parameters, i took note of asking, is it preferred `block.timestamp + x` or maybe set an initial date and an end date?

Reply:
let me talk to kingfish and will follow up